tests/integration/search/test_search_facets.py

```diff
@@ -74,19 +75,19 @@ def load_json_resource(resource):
 
 
 def test_hep_facets_by_author_name(api_client):
-    query = '"a Fecko"'
+    query = {'q': 'a Fecko'}
     expected = load_json_resource('test_hep_facets_by_author_name_expected_1')
 
     assert expected == json.loads(
-        api_client.get('literature/facets?q=%s' % query).data)
+        api_client.get('literature/facets?%s' % urllib.urlencode(query)).data)
 
-    query = '"a Müller"'
+    query = {'q': 'a Müller'}
     expected = load_json_resource('test_hep_facets_by_author_name_expected_2')
 
     assert expected == json.loads(
-        api_client.get('literature/facets?q=%s' % query).data)
+        api_client.get('literature/facets?%s' % urllib.urlencode(query)).data)
```

Comment:
```suggestion
        query = '"a Müller"'
        api_client.get('literature/facets?q=%s' % urllib.urlencode(query)).data)
```?

Reply:
urllib.urlencode take a dictionary with the query parameters as an argument, If I give it a string it throws an error